readme.txt

```diff
@@ -0,0 +1,36 @@
+=== Switch to ClassicPress from WordPress ===
+Contributors: classicpress
+Tags: classicpress, classic-press, classic, press
+Requires PHP: 5.2.4
+Requires at least: 4.9
+Tested up to: 4.9
+Stable tag: trunk
+License: GPLv2 or later
+License URI: https://www.gnu.org/licenses/gpl-2.0.html
+
+disable gutenberg call to action inside the dashboard
+
+== Description ==
+
+Disable Gutenberg call to action inside the dashboard.
+
+== Changelog ==
+
+= 1.0.2 =
+
+* Change slug and name according to ClassicPress community.
+
+= 1.0.1 =
```

Comment:
Please remove versions after 1.0.0 - we're still on the commits for the initial release

Reply:
OK to remove versions after v 1.0.0